[2022-01-30 17:30:54,984] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_dag.execution_date manual__2022-01-30T17:30:48.602318+00:00 [queued]>
[2022-01-30 17:30:55,027] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_dag.execution_date manual__2022-01-30T17:30:48.602318+00:00 [queued]>
[2022-01-30 17:30:55,029] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-30 17:30:55,030] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-01-30 17:30:55,031] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-30 17:30:55,079] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): execution_date> on 2022-01-30 17:30:48.602318+00:00
[2022-01-30 17:30:55,099] {standard_task_runner.py:52} INFO - Started process 1944 to run task
[2022-01-30 17:30:55,108] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_dag', 'execution_date', 'manual__2022-01-30T17:30:48.602318+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmph9von3__', '--error-file', '/tmp/tmpdac7qlqj']
[2022-01-30 17:30:55,112] {standard_task_runner.py:77} INFO - Job 41: Subtask execution_date
[2022-01-30 17:30:55,282] {logging_mixin.py:109} INFO - Running <TaskInstance: test_dag.execution_date manual__2022-01-30T17:30:48.602318+00:00 [running]> on host 51fb23ff9290
[2022-01-30 17:30:55,674] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_dag
AIRFLOW_CTX_TASK_ID=execution_date
AIRFLOW_CTX_EXECUTION_DATE=2022-01-30T17:30:48.602318+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-01-30T17:30:48.602318+00:00
[2022-01-30 17:30:55,690] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-30 17:30:55,694] {subprocess.py:74} INFO - Running command: ['bash', '-c', "echo 'execution date : 2022-01-30 ds_add: 2022-02-01'"]
[2022-01-30 17:30:55,727] {subprocess.py:85} INFO - Output:
[2022-01-30 17:30:55,792] {subprocess.py:89} INFO - execution date : 2022-01-30 ds_add: 2022-02-01
[2022-01-30 17:30:55,796] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-30 17:30:55,886] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_dag, task_id=execution_date, execution_date=20220130T173048, start_date=20220130T173054, end_date=20220130T173055
[2022-01-30 17:30:55,969] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-30 17:30:56,136] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
